### DISCUSSION TITLE
Mind map spread way out!

### DISCUSSION THREAD

**Discussion by dubtek** [freeplane/freeplane#1233] [Help]
I opened my main mind map and found the nodes spread way out the top and bottom. If I pull in the top, the bottom goes further out and vice versa. Fitting the whole map to the page makes everything miniscule. I can't find anything mentioned about such a thing. Help, please!
![Freeplane Expanded](https://github.com/freeplane/freeplane/assets/10489730/015a518e-36ce-4b3e-91df-8194bdbc3c83)

**Comment by dpolivaev** [comment]
Navigate->Select all visible nodes (⌘+A)
Edit->Reset node position (⌘+R)
